runtime/local/src/lib.rs

```diff
@@ -544,6 +547,173 @@ impl pallet_custom_signatures::Config for Runtime {
     type UnsignedPriority = EcdsaUnsignedPriority;
 }
 
+parameter_types! {
+    pub MaximumSchedulerWeight: Weight = NORMAL_DISPATCH_RATIO * RuntimeBlockWeights::get().max_block;
+    pub const MaxScheduledPerBlock: u32 = 50;
```

Comment:
I'd suggest you use `Const__<>` notation instead.

It's being migrated in parity's repos so we should also use it.